src/app/pages/app-detail/app-detail.component.ts

```
@@ -106,28 +108,30 @@ export class AppDetailComponent implements OnInit, AfterViewInit {
-          );
           this.initializeSwiper();
         }, 100);
       }
     });
+    this.subscriptions.push(langSub);
+  }
+
+  // Memory leak fix: Clean up all subscriptions on destroy
+  ngOnDestroy(): void {
+    console.log("🧹 DEBUG: Cleaning up subscriptions...");
+    this.subscriptions.forEach(sub => sub.unsubscribe());
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)